manager/notifier.go

```diff
@@ -325,6 +326,59 @@ func (n *notifier) sendSlackNotification(op notificationOp, config *pb.SlackConf
 	return nil
 }
 
+// https://www.flowdock.com/api/team-inbox
+// compulsory fields in Flowdock JSON: source, from_address, subject, content
+// Content-type: application/json
+// POST to https://api.flowdock.com/v1/messages/team_inbox/:flow_api_token
+type FlowdockMessage struct {
+	Source      string   `json:"source"`
+	FromAddress string   `json:"from_address"`
+	Subject     string   `json:"subject"`
+	Content     string   `json:"content"`
+	Format      string   `json:"format"`
+	Link        string   `json:"link"`
+	Tags        []string `json:"tags,omitempty"`
+}
+
+func jsonize(msg interface{}) []byte {
+	buf, err := json.Marshal(msg)
+	if err != nil {
+		glog.Errorln("Error compiling JSON:", err)
+	}
+	return buf
+}
+func getFlowdockNotificationMessage(op notificationOp, config *pb.FlowdockConfig, a *Alert) *FlowdockMessage {
```

Comment:
`get...` -> `new...` seems to be the naming convention in Go for functions which just build objects for you.